### ISSUE TITLE
DWW: add code to the portal to handle weakly positive expression as simple positive expression

### ISSUE THREAD

**Issue by drdiek** [Hippocampome-Org/php#8]
Originally, both positive and weakly-positive marker expression were displayed on the Neuron Page.  Charise kluged a solution by removing all weak expression from her spreadsheet before it was imported.  A better long term solution is to leave the database as is (i.e. it currently recognizes and stores weakly positive expression as being different from positive expression, which is good) and update the portal code so that it displays weakly positive expression as if it were just positive expression.  The code itself should not need to change much.  There are currently lines in there such as:
 
if ($variable == 'weak_positive')
// do something
 
if ($variable == ‘positive')
// do something different
 
if ($variable == ‘negative')
// do something different
 
So basically all we need to do is comment out the first if statement and make the second one into a logical OR:
 
if ($variable == ‘positive') || ($variable == ‘weak_positive')
 
The changes need to be made in several places (markers.php, neuron pages, evidence pages, search pages, etc.), so it will take a little time and require thoroughness.
